frontend/src/components/entity/style.ts

```diff
@@ -7,7 +7,7 @@ export const entityCardMainSx: SxProps = {
   border: '1px solid #DCD8CB',
   boxShadow: '0px 0px 8px rgba(0, 0, 0, 0.02), 0px 2px 4px rgba(0, 0, 0, 0.05)',
   borderRadius: '4px',
-  alignContent: 'flex-start',
+  alignContent: 'space-between',
```

Comment:
I am not really convinced by this change, but for now I can't find a better one :smile:

Reply:
Me too :upside_down_face: 

Do you know what you don't like precisely? On my side I was scared of unexpected visual side effects in the entity lists.

There is another way, maybe more "natural", to display the entity context alert at the bottom of entity cards: do not stretch the entity cards height.

 If we replace this layout...

![capture](https://github.com/tournesol-app/tournesol/assets/39056254/524136af-9ab9-44a8-b41d-389556e4fe17)

...by this layout...

![capture2](https://github.com/tournesol-app/tournesol/assets/39056254/38b1492c-03d1-4841-a341-85c4cf462a70)

...the entity context alert will be automatically displayed at the bottom. If I remember correctly we stretched the entity card on purpose to make them all visually equal. Reverting this could be seen as a "step backward".

Reply:
Another option could be to use `space-between` only when there is a context. I initially discarded this option because I didn't want to introduce more complexity in the way we define the CSS of entity cards. Nevertheless this can be a good "in between" solution.

Reply:
I submitted another approach in https://github.com/tournesol-app/tournesol/pull/1871